ansible/roles/kind-load-image/tasks/podman.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Check image exists locally
+  containers.podman.podman_image_info:
+    name: "{{ image }}"
+  register: image_info
+
+- name: Load image into kind cluster
+  ansible.builtin.command:
+    cmd: kind --name {{ kind_name }} load docker-image {{ image }}
+  when: image_info.images | length == 1
```

Comment:
since `image_info` should be available to `main.yml` after these tasks run we should be able to remove the duplication from the podman and docker files and put this task in main, but having said that, the implicit use of the image_info variable doesn't feel all that clean

Reply:
Are you sure about that? I initially tried adding 2 steps in the `k8s-integration-tests.yml` file that registered the `image_info` variable and when I skipped the second one it ended up being an empty variable.

Reply:
Actually, I'll give it a try.

Reply:
Ok, it works because the entire task file is skipped and so `image_info` is not registered twice.

I agree that using a variable that is registered "implicitly" from `main.yml`'s point of view is awkward, but I'd rather not have the uploading task duplicated, so I'm adding this change.